homeassistant/components/withings/coordinator.py

```diff
@@ -0,0 +1,299 @@
+"""Withings coordinator."""
+import asyncio
+from collections.abc import Callable
+from datetime import timedelta
+import logging
+from typing import Any
+
+from withings_api.common import (
+    AuthFailedException,
+    GetSleepSummaryField,
+    MeasureGroupAttribs,
+    MeasureType,
+    MeasureTypes,
+    NotifyAppli,
+    UnauthorizedException,
+    query_measure_groups,
+)
+
+from homeassistant.components.webhook import async_generate_path
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_WEBHOOK_ID
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.network import get_url
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+from homeassistant.util import dt as dt_util
+
+from .api import ConfigEntryWithingsApi
+from .const import Measurement
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Is there a reason why you use this logger here and the logger in const elsehwere?

Reply:
Made a preliminary PR for this